### PR TITLE
check gofmt with golang 1.4

### PR DESCRIPTION
**Pull request by eparis**

### CODE REVIEW

File: pkg/proxy/config/config.go [hunk 1]
```diff
@@ -262,7 +262,7 @@ func (s *serviceStore) MergedState() interface{} {
 // watchForUpdates invokes bcaster.Notify() with the latest version of an object
 // when changes occur.
 func watchForUpdates(bcaster *config.Broadcaster, accessor config.Accessor, updates <-chan struct{}) {
-	for _ = range updates {
+	for range updates {
```
Comment (vmarmol) [inline pkg/proxy/config/config.go]: Looks like Go 1.3 doesn't like this from the Travis breakage.

**Comment by eparis** [comment]
hmmm, so gofmt 1.4 with -l requires this change, but gofmt from 1.3 in shippable dies.  I have no idea how to handle this.  Just keep ignore people who use 1.4 ?

**Comment by eparis** [comment]
we could drop the -l check and then gofmt 1.4 would be happy with the old code....

**Comment by eparis** [comment]
-s is the problem, not -l.  whoops

**Comment by vmarmol** [comment]
LGTM, thanks @eparis! Will wait for green for merge

**Pull request merged**